recipes-kernel/linux/files/bcm2712-raspberrypi5-xen.dts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2024 EPAM systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+*/
+#include "bcm2712-rpi-5-b.dts"
+
+/ {
+	chosen {
+		#address-cells = <0x2>;
+		#size-cells = <0x2>;
+		xen,xen-bootargs = "console=dtuart dtuart=/soc/serial@7d001000 dom0_mem=256M dom0_max_vcpus=1 xsm=flask flask=permissive";
+		xen,dom0-bootargs = "console=hvc0 earlycon=xen earlyprintk=xen clk_ignore_unused root=/dev/ram0";
+		modules {
```

Comment:
This is incorrect as per Xen ARM booting.txt

- no need to have "module" node fro xen itself
- use correct compats only "multiboot,module" + [ "multiboot,kernel" | "multiboot,ramdisk" | "xen,xsm-policy"] 
- I don't know what is "xen-policy", so can't tell if it belongs to XEN or should belong to Domain. Most probably, it should belong to domain.

as result boot script is also incorrect.

Reply:
Fixed